src/Compilers/Core/Portable/CorLightup.cs

```diff
@@ -48,6 +48,138 @@ private static class CultureTypes
                     return null;
                 }
             }
+
+            private static class _Assembly
+            {
+                internal static readonly Type Type = typeof(Assembly);
+
+                internal static readonly Func<byte[], Assembly> Load_bytes = Type
+                    .GetTypeInfo()
+                    .GetDeclaredMethod("Load", typeof(byte[]))
+                    .CreateDelegate<Func<byte[], Assembly>>();
```

Comment:
Consider creating a helper for this series of calls.  It seems to be quite common.